var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -19,8 +19,10 @@ class PyNeurodamus(PythonPackage):
     version('2.4.0',   tag='2.4.0')
     version('2.3.1',   tag='2.3.1')
     version('1.3.2',   tag='1.3.2')
-    version('0.8.0',   tag='0.8.0')
-    version('0.7.2',   tag='0.7.2')
+
+    variant("all_deps",
```

Comment:
nitpicking - currently its only scipy dependency. do you envision more deps under this variant?

Reply:
Maybe yes. Like it happens with Spatial-Index, we may have more optional dependencies.
LibSonata was part of this group too but since it's quick to build I moved outside.